ports/unix/Makefile

```diff
@@ -1,8 +1,10 @@
 -include mpconfigport.mk
 include ../../py/mkenv.mk
 
-FROZEN_DIR = scripts
-FROZEN_MPY_DIR = modules
+# use FROZEN_MANIFEST for new projects, others are legacy
+FROZEN_MANIFEST = manifest.py
```

Comment:
While you're here, `?=` rather than `=`  (consistency with other ports)?

Reply:
Done